### PR TITLE
Editor's playbook for review.

### LINKED ISSUE #615: Specify the Editorial process (Note)
As a strawperson:
  * A majority of current editors need to actively agree for any non-normative change to a spec
  * All but one need to actively agree for any normative change to a spec
  * At least one (beyond the issuer of the PR, if applicable) for any change to non-spec content on the site

Changes to specifications:
  * MUST have a documented use case, supported by at least two institutions
  * MUST have two server-side implementations, at least one of which SHOULD be production ready
  * MUST have at least one client-side implementation, which may be a proof of concept

### PR DESCRIPTION
**Pull request by jpstroop**
Resolves #615. I will squash commits when we're all :+1:

### CODE REVIEW

**Comment by jpstroop** [comment]
Oh, and to follow the new rules: http://editors.iiif.io/api/annex/notes/editors.html :wink:

**Comment by azaroth42** [comment]
I've read and have only micro level wording suggestions. Assuming wifi on flight, I'll branch and PR them to this branch?

**Comment by jpstroop** [comment]
Why not do line comments on the latest diff on the PR, and the pull that
same (editors) branch, make the changes, and push it back up?

-Js

Sent via mobile. Please excuse typos, brevity, etc.

File: source/api/annex/notes/editors.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+----
+
+## Contents
+{:.no_toc}
+* Table of Discontent (will be replaced by macro)
+{:toc}
+
+## 1. Overview
+
+The IIIF editors are responsible for facilitating discussions across the IIIF community, digesting those discussions into use cases, and reflecting those use cases into new and revised specifications for the community to implement. This process is intended to be as transparent as possible. This document exists to further that intention by defining the IIIF editorial process and the responsibilities and expectations of the individual editors.
```
Comment (azaroth42) [inline source/api/annex/notes/editors.md]: > , ensuring those discussions are captured as use cases, 
as we don't want to be responsible for writing up all of the use cases, just making sure they're captured.

File: source/api/annex/notes/editors.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+## 2. Community Process
+
+### 2.1 Proposals and Feedback
+
+Community members should propose and discuss features and changes via [IIIF-Discuss][iiif-discuss]. However, there are a few other ways in which the IIIF Community can reach out to the editors. In particular, the IIIF holds a bi-weekly community conference call, the details and agenda for which are announced on IIIF-Discuss. Additionally, the [iiif.io issues on Github][iiif-github-issues] are open, and are especially useful for implementers who want to track the editorial discussion of a specific issue. Finally, conversations with individual editors are also possible, but are the least practical way to propose changes because the conversation will then have to be repeated to allow community input through one or more of the channels listed above.
```
Comment (azaroth42) [inline source/api/annex/notes/editors.md]: > the IIIF Community can engage in the specification process.

Reach out makes us seem remote and elite.

> track the discussion around an issue of interest.

Want to encourage non editorial discussion

File: source/api/annex/notes/editors.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+### 2.2 Use Cases and Support
+
+New specifications and changes to existing specification must have one or more [documented use cases][use-cases], supported by at least two institutions that either have IIIF-based technology in production or are clearly blocked from doing so, as demonstrated by the use case. In the case of breaking changes, where applicable, the documented use case must clearly demonstrate why the previous approach was flawed or insufficient.
+
+### 2.3 Evaluation and Testing
+
+In order to be considered ready for review, new features must have two open-source server-side implementations, at least one of which should be in production. New features must also have at least one open-source client-side implementation, which may be a proof-of-concept.
```
Comment (azaroth42) [inline source/api/annex/notes/editors.md]: > considered ready for final review, as below, 

Implementations should not precede proposals and general review and draft spec writing.

File: source/api/annex/notes/editors.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+### 2.4 Community Review
+
+New versions of specifications at or above 1.0 must be reviewed at an open meeting, to which the community is invited. These meetings will be announced on [IIIF-Discuss][iiif-discuss], [IIIF-Announce][iiif-announce] and other relevant community email lists. Such meetings must occur at least once per year, and should occur twice. The specifications will be frozen two weeks prior to a review meeting, and this review period will be announced to the same lists as above.
```
Comment (azaroth42) [inline source/api/annex/notes/editors.md]: I will just note that LDCX is not an open meeting, and hence doesn't qualify, even if all eds are going to be there :crying_cat_face: /cc @tomcramer

File: source/api/annex/notes/editors.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+
+### 3.1 Terminology
+
+The key words _MUST_, _MUST NOT_, _REQUIRED_, _SHALL_, _SHALL NOT_, _SHOULD_, _SHOULD NOT_, _RECOMMENDED_, and _OPTIONAL_ in this document are to be interpreted as described in [RFC 2119][rfc-2119].
+
+### 3.2 Process for Suggesting Changes
+
+Editorial collaboration takes place primarily in the [iiif.io repository on GitHub][iiif-github]. When editors submit changes for revision, they _MUST_ adhere to the following criteria:
+
+ * There _SHOULD_ be a Github issue that explains the reason for the change and serves as a platform for discussion. Discussion regarding smaller changes may take place directly on a pull request.
```
Comment (azaroth42) [inline source/api/annex/notes/editors.md]: smaller, primarily wording level, changes

File: source/api/annex/notes/editors.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+ * __All but one__ of the editors _MUST_ actively agree on any __normative__ change to a specification.
+ * Even for non-specification content, __at least one__ editor, in addition to the issuer of the pull request, if applicable, must actively agree to any change on the [iiif.io][iiif-io] website.
+
+The editor that submitted the pull request _MUST NOT_ be the editor that merges it.
+
+In the event that an editor disagrees with a merge that has already happened, they should create a new issue that references the change in question, and explains the objection. This issue then serves as the discussion point for the objection which is subject to this same process. Reverts _SHOULD_ stay in the repository history, i.e., the process of reverting involve updating the HEAD version to reflect older content, and not erasing or re-writing history.
+
+### 3.4 Face to Face Meetings
+
+Editors will meet at least four times per year, and all editors _MUST_ attend at least three of those meetings per calendar year. Every effort will be made to co-locate at least one of the meetings with another meeting at which most of editors are already likely to be present.
```
Comment (azaroth42) [inline source/api/annex/notes/editors.md]: I thought we were going for at least three and MUST attend two? Four and three is a pretty high barrier for international travel

File: source/api/annex/notes/editors.md [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+
+The editors _MUST_ provide a summary of all meetings, and _SHOULD_ provide a summary of other interactions (e.g. closed conference calls leading up to a specification review period) at which changes and new specifications are discussed.
+
+## 4. Editorial Team Membership and Selection
+
+Per the IIIF Memorandum of Understanding covering the establishment of an IIIF Consortium, the editors may nominate additional editors at any time. All current editors _MUST_ agree before an invitation is issued. Editors will chose from participants within the community and/or experts willing to lend their knowledge and experience to new specifications. Opportunities to better the gender, race, and age balance of the existing editorial team will also be a consideration. New editors _MUST_ be approved by the IIIF Executive Group.
```
Comment (azaroth42) [inline source/api/annex/notes/editors.md]: > Understanding that established the IIIF Consortium,

Comment (azaroth42) [inline source/api/annex/notes/editors.md]: I think we also agreed that candidate editors __SHOULD__ have made successful changes via github, following the protocol defined.

File: source/api/annex/notes/editors.md (hunk 6 above)
Comment (jpstroop) [inline source/api/annex/notes/editors.md]: I think I cut and pasted this from our slack or github conversation. I'm :OK: with either.

**Comment by azaroth42** [comment]
Proposed changes above. Only substantive one is 3 of 4 meetings, down to 2 of 3. It is _at least_ so we can choose to meet more often

File: source/api/annex/notes/editors.md (hunk 6 above)
Comment (azaroth42) [inline source/api/annex/notes/editors.md]: @IIIF/editors Thoughts?

File: source/api/annex/notes/editors.md (hunk 4 above)
Comment (jpstroop) [inline source/api/annex/notes/editors.md]: I was assuming days like the Wednesday meetings we've had following the outreach events would serve this purpose.

File: source/api/annex/notes/editors.md (hunk 2 above)
Comment (jpstroop) [inline source/api/annex/notes/editors.md]: Yes, but not fork community discussion from iiif-discuss

File: source/api/annex/notes/editors.md (hunk 4 above)
Comment (azaroth42) [inline source/api/annex/notes/editors.md]: Comment now hidden in outdated diff ... Not sure I like line notes ... And yes, agreed re day-after-event-days ... but that may limit us to twice a year ... which is probably okay.

File: source/api/annex/notes/editors.md (hunk 3 above)
Comment (jpstroop) [inline source/api/annex/notes/editors.md]: It occurs to me now that this is really about new versions of specs. We can put something into version n+1 draft, but we can't ratify a new version of a spec until the changes have reference implantations.

File: source/api/annex/notes/editors.md (hunk 2 above)
Comment (azaroth42) [inline source/api/annex/notes/editors.md]: Agreed.
I think we need to establish some norms around iiif-discuss being *the* place for community discussion. I have grave concerns re manuscript interest group.

File: source/api/annex/notes/editors.md [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+
+### 2.2 Use Cases and Support
+
+New specifications and changes to existing specification must have one or more [documented use cases][use-cases], supported by at least two institutions that either have IIIF-based technology in production or are clearly blocked from doing so, as demonstrated by the use case. In the case of breaking changes, where applicable, the documented use case must clearly demonstrate why the previous approach was flawed or insufficient.
+
+### 2.3 Evaluation and Testing
+
+In order to be considered ready for final review, as below, new features must have two open-source server-side implementations, at least one of which should be in production. New features must also have at least one open-source client-side implementation, which may be a proof-of-concept.
+
```
Comment (jpstroop) [inline source/api/annex/notes/editors.md]: (Copying my comment over from the outdated diff): It occurs to me now that this is really about new versions of specs. We can put something into version n+1 draft, but we can't ratify a new version of a spec until the changes have reference implementation..

**Comment by jpstroop** [comment]
@IIIF/editors can __everyone__ review? Are we close with this?

**Comment by jpstroop** [comment]
@IIIF/editors I made [three minor changes](https://github.com/IIIF/iiif.io/commit/6d5456bbbfb042bc1dd2b2f4da9098d0ac1b8096)

Read here: http://editors.iiif.io/api/annex/notes/editors.html

Can we make this happen before the holidays :dog: :horse: ?

**Comment by azaroth42** [comment]
:+1: from me as is, with suggestions below.

I think the 3/4 meetings is okay for now and for me personally (e.g. 2016), but might be hard to sustain.  Assuming we have one big and one small IIIF event each year, swapping between Europe and US, plus an editor specific meeting, that leaves one more (LDCX and/or OR, given current practice).  But already there are conflicts, no? Simeon is out for quite a while over summer, Jon has a conflict for the current May dates.  We're not going to kick people out for missing the target by one, so we might as well encode that in the doc:

The editors will meet at least three times per year (with the intention of meeting more often), and all editors must attend at least two of those meetings. The meeting time and date will be agreed upon within the editorial group, and every effort ...

**Comment by jpstroop** [comment]
Other opinions? I feel confident about being able to commit to 3/4. We might be able to make it work so that between the annual European event and OR that there are generally two meetings in Europe as well (could always find another meeting to co-locate with as well). 

I'm there's a preference to loosen the language or say half, I'm OK with that too.

**Comment by jpstroop** [comment]
Typed with my thumb... You the gist. 😉

**Comment by zimeon** [comment]
I'm also OK with 3/4 meetings. We can change it if it doesn't work. I made a couple of small typo fixes in b61bbee but also:
  * I added a source/_includes/copyright2015.md for copyright starting in 2015. It seems silly to put up new documents that say 2012--. Now, this does look silly right now as it come out as 2015-2015 but that will be fixed in a < 2 weeks by the steady progression of time
  * I squashed the change log to one entry saying "2015-12-22 First release (@mikeapp, @tomcrane, @azaroth42, @jpstroop, @zimeon)" ... just in case we can agree by tomorrow

I'm :+1: now and you can carry that forward through any minor changes I might not response to in the next day or so

**Comment by tomcrane** [comment]
I am OK with 3/4 or 4/4 meetings in 2016 given that I am fairly clear on where 3 of those will be. As @zimeon says we can change this if we get to the end of 2016 and another 3/4 meetings in 2017 look difficult. So :+1:

**Comment by mikeapp** [comment]
I'm OK with 3/4 also, given the currently planned meetings.      :+1:

**Comment by jpstroop** [comment]
OK @IIIF/editors, my PR, so whenever someone else wants to pull the trigger.

**Pull request merged**